### ISSUE TITLE
Add SVG opacity properties as exceptions to alpha-value-notation

### ISSUE THREAD

**Issue by Mouvedia** [stylelint/stylelint-config-standard#257]
### What steps are needed to reproduce the bug?

use one of these SVG properties:
- [flood-opacity](https://drafts.fxtf.org/filter-effects/#FloodOpacityProperty)
- [fill-opacity](https://drafts.fxtf.org/fill-stroke/#fill-opacity) (also in [SVG2](https://svgwg.org/svg2-draft/painting.html#FillOpacity))
- [stroke-opacity](https://drafts.fxtf.org/fill-stroke/#stroke-opacity) (also in [SVG2](https://svgwg.org/svg2-draft/painting.html#StrokeOpacity))
- stop-opacity (in [SVG2](https://svgwg.org/svg2-draft/pservers.html#StopOpacityProperty))

ref https://github.com/w3c/csswg-drafts/issues/3342

### What did you expect to happen?

to be exceptions as well

### What actually happened?

ref #210
see also https://test.csswg.org/harness/details/css-color-3_dev/opacity-computed/

**Comment by jeddy3** [comment]
@Mouvedia Thanks for the issue and for providing all the references.

The rule currently [only checks the `opacity` and `shape-image-threshold`](https://github.com/stylelint/stylelint/blob/4772851c0a1745db03f149ac35a2de6fe375c369/lib/rules/alpha-value-notation/index.js#L26). We can add the properties above to the rule and then as exceptions here in the config.

Do you want to open a false negatives issue for the properties above in Stylelint?

**Comment by Mouvedia** [comment]
> Do you want to open a false negatives issue for the properties above in Stylelint?

If both releases of stylelint-config-standard and stylelint are synchronized, yes.
Else Id prefer if the exceptions were added beforehand—even if they are _inactive_ for a short while.

**Comment by jeddy3** [comment]
> If both releases of stylelint-config-standard and stylelint are synchronized, yes.

Yes, [they're synced](https://stylelint.io/maintainer-guide/releases). We do new releases of the configs, if we make any changes to Stylelint that impact either of them.

**Comment by Mouvedia** [comment]
> Yes, [they're synced](https://stylelint.io/maintainer-guide/releases).

Alright then. Ill either open an issue or a PR.

**Comment by Mouvedia** [comment]
Do we really want to have 5 exceptions out of 6 properties targetted?
i.e. it only prefers `percentage` for `shape-image-threshold`
```
		'alpha-value-notation': [
			'percentage',
			{
				exceptProperties: [
                	'opacity', 
                	'fill-opacity',
                	'flood-opacity',
                	'stop-opacity',
                	'stroke-opacity',
               ],
			},
		],
```
I think the saner option should be to just pick `number` as the default.
When it's 5 out of 6 it's not really exceptions anymore…
@jeddy3 @ntwb

**Comment by jeddy3** [comment]
We can remove each exception when it has fuller browser support. Percentages feel more natural, especially in colours, e.g. `hsl(0deg 100% 50% / 25%)`, which I assume is why they've often been used in graphics apps and been added to the language.
